src/meta/processors/admin/SnapShot.cpp

```diff
@@ -92,7 +92,7 @@ StatusOr<std::map<GraphSpaceID, std::set<HostAddr>>> Snapshot::getSpacesHosts()
         LOG(ERROR) << "Get hosts meta data error";
         return Status::Error("Get hosts meta data error");
     }
-    while (iter->valid()) {
+    for (; iter->valid(); iter->next()) {
```

Comment:
What's the difference? I don't understand.

Reply:
the continue will loop forever